README.md

```diff
@@ -11,56 +11,66 @@ developer guide](http://pravega.io).
-| `turbineheatprocessor` | A Flink streaming application for processing temperature data from a Pravega stream produced by the `turbineheatsensor` app. The application computes a daily summary of the temperature range observed on that day by each sensor. | [Java](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/java/io/pravega/examples/flink/iot), [Scala](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/scala/io/pravega/examples/flink/iot)
-| `anomaly-detection` | A Flink streaming application for detecting anomalous input patterns using a finite-state machine. | [Java](https://github.com/pravega/pravega-samples/tree/master/anomaly-detection)
+| [`turbineheatsensor`](https://github.com/pravega/pravega-samples/tree/master/scenarios/turbine-heat-sensor) | It emulates parallel sensors producing temperature values (writers) and parallel consumers performing real-time statistics (readers) via Pravega client. | [Java](https://github.com/pravega/pravega-samples/tree/master/scenarios/turbine-heat-sensor/src/main/java/io/pravega/turbineheatsensor)
+| [`turbineheatprocessor`](https://github.com/pravega/pravega-samples/tree/master/scenarios/turbine-heat-processor) | A Flink streaming application for processing temperature data from a Pravega stream produced by the `turbineheatsensor` app. The application computes a daily summary of the temperature range observed on that day by each sensor. | [Java](https://github.com/pravega/pravega-samples/tree/master/scenarios/turbine-heat-processor/src/main/java/io/pravega/turbineheatprocessor), [Scala](https://github.com/pravega/pravega-samples/tree/master/scenarios/turbine-heat-processor/src/main/scala/io/pravega/turbineheatprocessor)
+| [`anomaly-detection`](https://github.com/pravega/pravega-samples/tree/master/scenarios/anomaly-detection) | A Flink streaming application for detecting anomalous input patterns using a finite-state machine. | [Java](https://github.com/pravega/pravega-samples/tree/master/scenarios/anomaly-detection/src/main/java/io/pravega/anomalydetection)
 
 
 # Build Instructions
 
+
+The `pravega-samples` project is prepared for working with [release artifacts](https://github.com/pravega/pravega/releases) of Pravega components, 
+which are already available in Maven central. Moreover, you also have the option to configure `pravega-samples`
+to work with `master` snapshots artifacts published in our [JFrog repository](https://oss.jfrog.org/artifactory/jfrog-dependencies/io/pravega/).
+You can easily do this by specifying the desired artifact version (`pravegaVersion`, `flinkConnectorVersion`,
+`hadoopConnectorVersion`) in the `gradle.properties` file. Thus, if you look for a quick start 
+you may skip the _optional_ sections; otherwise, if you want to have fun building the different projects from source,
+please read the next sections. 
+
+
 ## Pre-requisites
 
 * Java 8+
```

Comment:
I don't think we support Java 9, although we should by now.

Reply:
At the moment, I set `Java 8`.